app/client/stylesheets/main.scss

```diff
@@ -189,8 +194,13 @@ html {
 }
 
 .right-float {
-  float: right;
+  text-align: right;
   clear: none;
+  float: right;
```

Comment:
In the diff view here, you've removed `float: right;` and then added it. That makes this review slightly harder to read -- in particular, it makes it more difficult to determine your intent.

This is not a showstopper, just a remark.